tests/python/unittest/test_numpy_op.py

```diff
@@ -1080,6 +1080,80 @@ def hybrid_forward(self, F, a, *args):
                 assert same(mx_out.asnumpy(), np_out)
 
 
+@with_seed()
+@use_np
+def test_np_degandrad():
+    class TestDeg2rad(HybridBlock):
+        def __init__(self):
+            super(TestDeg2rad, self).__init__()
+
+        def hybrid_forward(self, F, x):
+            return F.np.deg2rad(x)
+
+    class TestRad2deg(HybridBlock):
+        def __init__(self):
+            super(TestRad2deg, self).__init__()
+
+        def hybrid_forward(self, F, x):
+            return F.np.rad2deg(x)
+
+    types = ['float64', 'float32']
+    for hybridize in [True, False]:
+        for shape in [(),
+                      (1,),
+                      (1, 1),
+                      (1, 2, 3),
+                      (1, 0),
+                      (2, 0, 3)
+                      ]:
```

Comment:
simply do
```Python
for shape in [(), (1,), (1, 1), (1, 2, 3), (1, 0), (2, 0, 3)]:
```

Reply:
Have put the test into test_np_unary_funcs. Thanks.